README.rst

```diff
@@ -56,19 +56,17 @@ To install in a virtual environment in your current project:
 Usage Example
 =============
 
-.. code-block:: python
+.. code-block:: python3
 
     import time
     import board
-    import busio
     import adafruit_lis2mdl
 
-    i2c = busio.I2C(board.SCL, board.SDA)
+    i2c = board.I2C()
```

Comment:
Should this include the `uses board.SCL and board.SDA` comment you included with the others?

Reply:
@kattni  I could included here, in the newer libraries, meaning they just create the object without the comment so I tried to keep the same style as the library, no problem to add it, let me know

Reply:
Changes in https://github.com/adafruit/Adafruit_CircuitPython_LIS2MDL/pull/13/commits/a5a8dfb3113f35f419452c2715b7ff99037e39c4